card/mod/basic_formats/set/all/json.rb

```diff
@@ -29,30 +27,6 @@ def show view, args
     JSON.send method, raw
   end
 
-  # TODO: design better autocomplete API
-  # view :name_complete, cache: :never do
-  #   name_search
-  # end
-
-  view :name_complete, cache: :never do
-    name_search query_attribute: :junction_complete
-  end
```

Comment:
these views depend on `card.search`, which not all cards can access.  That's why I moved them into abstract/search.